cypress/support/pages/PlaceAnOrder.pageObject.js

```diff
@@ -0,0 +1,123 @@
+import PageObject from '../PageObject';
+
+class PlaceOnOrderPageObject extends PageObject {
+  url = '/index.html';
+
+  get emailField() {
+    return cy.get('#email');
+  }
+
+  get nameField() {
+    return cy.get('#name');
+  }
+
+  get countryField() {
+    return cy.get('#country');
+  }
+
+  get cityField() {
+    return cy.get('#city');
+  }
+
+  get messageAllert() {
```

Comment:
There is a typo in the method name 'messageAllert'. It should be 'messageAlert'.